app/controllers/revisions_controller.rb

```diff
@@ -54,28 +62,37 @@ def load_node
     initialize_nodes_sidebar
   end
 
-  def load_record
-    @record = if params[:card_id]
-                @list.cards.find(params[:card_id])
-              elsif params[:evidence_id]
-                @node.evidence.find(params[:evidence_id])
-              elsif params[:issue_id]
-                Issue.find(params[:issue_id])
-              elsif params[:note_id]
-                @node.notes.find(params[:note_id])
-              else
-                raise 'Unable to identify record type'
-              end
-  rescue ActiveRecord::RecordNotFound
-    flash[:error] = 'Record not found'
-    redirect_to :back
+  def set_revision_params
+    @revision_params = { :"#{@record.class.name.downcase}_id" => @record.id }
```

Comment:
```suggestion
    @revision_params = { "#{@record.class.name.downcase}_id" => @record.id }
```

not sure if we need this to be a symbol vs a string